Classes/Command/FluidPrecompilerServiceTrait.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace NamelessCoder\CmsFluidPrecompilerModule\Command;
+
+use NamelessCoder\CmsFluidPrecompilerModule\Service\FluidPrecompilerService;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+use TYPO3\CMS\Extbase\Object\ObjectManager;
+
+/**
+ * Trait used by commands to get the FluidPrecompilerService
+ */
+trait FluidPrecompilerServiceTrait
```

Comment:
I'm not sure we really need this trait; this could also be done with an abstract command which provides this getter.

Reply:
Duplicating the getter in two classes is also perfectly acceptable if the alternative is to create a single-method trait or abstract class. In fact, I'd say duplication is preferred for single-line, single-function cases.